### ISSUE TITLE
2D Dataset train

### ISSUE THREAD

**Issue by fm52Hz** [ziyangwang007/Mamba-UNet#31]
Hello，I would like to test your code with my dataset, but all the datasets you currently provide are 3D. I'm working with 2D PNG images and have modified the` BaseDataSets` code to return data that fits the format `volume_batch, label_batch = sampled_batch['image'], sampled_batch['label'].`
However, during training, the loss drops to zero around iteration 200, and the validation loss is also zero. I'm not sure how to handle 2D images to make the model run correctly.
Could you please give me some advice on how to process 2D images so that the model can function properly?
